src/cc_catalog_airflow/dags/provider_api_scripts/common/licenses/constants.py

```diff
@@ -0,0 +1,138 @@
+NO_VERSION = 'N/A'
+
+_SIMPLE_LICENSE_PATHS = [
+    # This list holds recognized license URL path snippets that split
+    # correctly into license_ and license_version, and for which we can
+    # recover the path from a valid license_, license_version pair.
```

Comment:
These snippets are substrings of the missing ported license paths.  We'll never throw away the port information if it is given as input.  The idea is we can recover the license, version info that's used in the UI (see my other comment) regardless of the port, if we see one of these substrings.

These are reversible, since we can 'reverse map' a (license, version) pair to the unported versions (or the Japanese version in the case where there's no unported version and only a Japanese port).